src/web/mjs/engine/DownloadManager.mjs

```diff
@@ -36,12 +37,53 @@ export default class DownloadManager extends EventTarget {
         }
     }
 
+    /**
+     * Returns true if the download queue is paused.
+     */
+    isQueuePaused() {
```

Comment:
May use a getter instead of a function.

Reply:
i'd also prefer having a "stratus" field instead of "isPaused".
List of status in haruneko: https://github.com/manga-download/haruneko/blob/master/web/src/engine/DownloadTask.ts#L6